src/testing/mod.rs

```diff
@@ -0,0 +1,20 @@
+pub mod rest_client;
+pub mod server;
+pub mod startup;
+
+use crate::{db::models::api_tokens::APITokenData, v0::api_token::APIToken};
+
+use chrono::offset::Utc;
+
+pub fn get_testing_token() -> (APITokenData, String) {
```

Comment:
It’s ok for now here. But I tried to keep the Sam structure in the project so it i easy to  locate everything. Maybe we can have a sub module for this kind of calls (testing-data?). Inside a token file where this would be. Even if we make it public from the mod later, so we keep it easy to locate and offer a simple way and an explicit way of importing it. What do you think?

Reply:
agree. I've put it here because it didn't want to make module for only one method. But  this is a good idea to have separate module for test data